### PR TITLE
New GaGs clothing: Part 1 - 'Dress'ing up!

### PR DESCRIPTION
**Pull request by xXPawnStarrXx**
## About The Pull Request
I'm going to keep this simple, I have added four GaGs configs and the sprites for some outfits, I intend to expand on the recolouring abilities, so you will be able to use more than just spraycans.
The outfits are as follows; The Cardigan skirt (formerly named black skirt), the Evening gown (It's no longer the 'RED' gown, since it can be any colour), the striped dress and the sailor dress.

image
image
image
image
## Why It's Good For The Game
Customisation is always good, allowing players to express their character how they want isn't a bad thing!
## Changelog
:cl:
image: added gags sprites for; Sailor dress, Evening gown, Cardigan skirts and striped dresses.
/:cl:

### CODE REVIEW

**Comment by xXPawnStarrXx** [comment]
I'll try see why it's failing checks tomorrow. My brain is sleepy.

**Comment by Wallemations** [comment]
fix your broken checks first

**Review by Wallemations — APPROVED** [review]

File: code/modules/clothing/under/skirt_dress.dm [hunk 1]
```diff
@@ -27,17 +46,35 @@
 	body_parts_covered = CHEST|GROIN|LEGS
 	flags_inv = HIDESHOES
 
-/obj/item/clothing/under/dress/redeveninggown
```
Comment (Wallemations) [inline code/modules/clothing/under/skirt_dress.dm:30]: Did any of the maps have the evening gown in them? I always sucked at reading failed checks but it seems they're mostly map errors with creating/destroying things that aren't there.

Comment (xXPawnStarrXx) [reply code/modules/clothing/under/skirt_dress.dm:30]: I'll check when I'm home on the weekend, away for work for a few more days.

Comment (jughu) [reply code/modules/clothing/under/skirt_dress.dm:30]: Delta has one

Comment (xXPawnStarrXx) [reply code/modules/clothing/under/skirt_dress.dm:30]: Noted! Thank you. I'll go through all the .dmm files and check tomorrow too.

**Comment by xXPawnStarrXx** [comment]
Gotta work out why "/datum/unit_test/suit_storage_icons", "/datum/unit_test/modify_fantasy_variable" and "/datum/unit_test/create_and_destroy" are failing.

**Comment by Wallemations** [comment]
Could ask a code maint

**Comment by xXPawnStarrXx** [comment]
> Could ask a code maint

I finally fixed it with Melbert's help.

**Pull request merged**